### DISCUSSION TITLE
Not able to start minikube cluster

### DISCUSSION THREAD

**Discussion by kapilmalik1983** [unfor19/kubernetes-localdev#3] [Q&A] [answered]
While I am trying to start cluster on WSL, I am getting docker error. I have verified on windows machine that docker ostype is set to Linux.
Under system tray on windows system, it says that docker desktop is running. What could I be missing?
Do I need to install docker on WSL too or do I need to install minikube on windows?

++++++++++++++++++
root@DESKTOP-HI3K193:~# minikube start --driver=docker --kubernetes-version=v1.20.2
😄  minikube v1.19.0 on Ubuntu 20.04
✨  Using the docker driver based on user configuration

🤷  Exiting due to PROVIDER_DOCKER_NOT_FOUND: The 'docker' provider was not found: exec: "docker": executable file not found in $PATH
💡  Suggestion: Install Docker
📘  Documentation: https://minikube.sigs.k8s.io/docs/drivers/docker/

root@DESKTOP-HI3K193:~#
+++++++++++++++++++

PS C:\Users\lenovo> docker info --format '{{.OSType}}'
linux
PS C:\Users\lenovo>
+++++++++++++++++++

**Comment by unfor19** [comment, marked as answer]
Docker for Desktop on Windows enables mounting the Docker daemon automatically in WSL2.

Make sure that you're running `minikube` on WSL2, after following this tutorial - https://docs.docker.com/docker-for-windows/wsl/

Let me know if you still have issues

**Reply by kapilmalik1983** [reply]
Thanks for the response. It worked after reinstalling Docker on windows. 
After reading the article, I found that docker needs to be installed after WSL

**Reply by unfor19** [reply]
@kapilmalik1983 Let me know if you need anything else, if not, please mark this question as answered

**Reply by kapilmalik1983** [reply]
Marked, thanks again !!
